access/pagerduty/config.go

```diff
@@ -15,18 +10,22 @@ import (
 type Config struct {
 	Teleport  lib.TeleportConfig `toml:"teleport"`
 	Pagerduty PagerdutyConfig    `toml:"pagerduty"`
-	HTTP      lib.HTTPConfig     `toml:"http"`
 	Log       logger.Config      `toml:"log"`
 }
 
 type PagerdutyConfig struct {
-	APIEndpoint string `toml:"-"`
-	APIKey      string `toml:"api_key"`
-	UserEmail   string `toml:"user_email"`
-	ServiceID   string `toml:"service_id"`
-	AutoApprove bool   `toml:"auto_approve"`
+	APIEndpoint        string `toml:"-"`
+	APIKey             string `toml:"api_key"`
+	UserEmail          string `toml:"user_email"`
+	RequestAnnotations struct {
+		NotifyService string `toml:"notify_service"`
+		Services      string `toml:"services"`
+	}
 }
 
+const NotifyServiceDefaultAnnotation = "pagerduty_notify_service"
+const ServicesDefaultAnnotation = "pagerduty_services"
+
 const exampleConfig = `# example teleport-pagerduty configuration TOML file
 [teleport]
 auth_server = "example.com:3025"                            # Teleport Auth Server GRPC API address
```

Comment:
example config should show how to use identity files in addition to individual cert files. Same with the config in the README.